cogs/moderation.py

```diff
@@ -86,6 +86,11 @@ async def report(self, ctx, user: discord.Member, *reason):
                 
                 print(error)
                 return await ctx.send(embed=discord.Embed(title="Error!", description="Error making the report! Please try again."))
+            
+    @commands.Cog.listener(name='on_message')
```

Comment:
Can you add some extra documentation under the function, just cleans up the code a bit.

Reply:
Submitted changes.